p2p/pex/pex_reactor.go

```diff
@@ -338,9 +341,22 @@ func (r *PEXReactor) ensurePeers() {
 		toDial[try.ID] = try
 	}
 
+	// 1s == (1e9 ns) == (1 Billion ns)
+	billionNs := float64(time.Second.Nanoseconds())
+
 	// Dial picked addresses
-	for _, item := range toDial {
+	for _, addr := range toDial {
 		go func(picked *p2p.NetAddress) {
```

Comment:
With `ensurePeers` growing further maybe we can break this function out and make it a private method on `PEXReactor` as it only seemed to depend on the reactor itself and the `*p2p.NetAddress`.